### PR TITLE
added new method DeleteUser

### PR DESCRIPTION
**Pull request by shelkon**

### CODE REVIEW

File: MedicalDocManagment.WebUI/Controllers/api/AdminController.cs [hunk 1]
```diff
@@ -104,6 +104,25 @@ private IHttpActionResult GetErrorResult(IdentityResult result)
             var result = await UsersManager.UpdateAsync(userInDb);
             return Ok(result);
         }
-        
+        [HttpDelete]
+        public async Task<HttpResponseMessage> DeleteUser(int id)
```
Comment (KostetskyiYaroslav) [inline MedicalDocManagment.WebUI/Controllers/api/AdminController.cs]: In our's project the `id` property isn't integer variable, it is string

File: MedicalDocManagment.WebUI/Controllers/api/AdminController.cs [hunk 2]
```diff
@@ -104,6 +104,25 @@ private IHttpActionResult GetErrorResult(IdentityResult result)
...
+            var user = await UsersManager.FindByIdAsync(id.ToString());
+            if (user != null)
+            {
+                if (!user.IsActive)
+                {
+                    return Request.CreateErrorResponse(HttpStatusCode.BadRequest, new HttpError("User already deleted."));
+                }
+                user.IsActive = false;
+                var result = await UsersManager.UpdateAsync(user);
+                if (result.Succeeded)
```
Comment (KostetskyiYaroslav) [inline MedicalDocManagment.WebUI/Controllers/api/AdminController.cs:119]: When `result` is going to be `false` the method will be return `Not Found`. It isn't critical bug, but anyway. You may use Alexander's method ( `GetErrorResponse` )

**Review by KostetskyiYaroslav — COMMENTED** [review]
I left two comments for some piece of code ( First one critical, another no )

**Review by AndriiKatsubo — APPROVED** [review]

**Review by KostetskyiYaroslav — APPROVED** [review]
ok

**Pull request merged**